code/stable/projectile/Lesson/Projectile_Lesson.ipynb

```diff
@@ -394,11 +394,10 @@
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
```

Comment:
I'm fine with removing `CodeBlock`, but I am not fine with "cell_type" changing from "code" to "markdown".

Reply:
Let's keep thinking about other solutions to have certain things end up in the Jupyter document marked as 'code'. For example, a wrapper around `Expr` instead of being *inside* `Expr` might do. All we really need is enough information to emit a piece of code in the back-end.